### PR TITLE
Add ddoc example to isImplicitlyConvertible

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

**Comment by burner** [comment]
LGTM

File: std/traits.d [hunk 1]
```diff
@@ -4137,20 +4137,18 @@ template isImplicitlyConvertible(From, To)
...
-
     static assert(!isImplicitlyConvertible!(wchar, char));
 
-    // bug6197
     static assert(!isImplicitlyConvertible!(const(ushort), ubyte));
     static assert(!isImplicitlyConvertible!(const(uint), ubyte));
     static assert(!isImplicitlyConvertible!(const(ulong), ubyte));
 
-    // from std.conv.implicitlyConverts
     assert(!isImplicitlyConvertible!(const(char)[], string));
```
Comment (burner) [inline std/traits.d]: on second look why is this an assert not an static assert?

**Comment by JackStouffer** [comment]
@burner fixed

**Comment by burner** [comment]
that was fast

**Comment by burner** [comment]
Auto-merge toggled on

**Comment by JackStouffer** [comment]
Thanks!

**Comment by burner** [comment]
thank you

**Pull request merged**